package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remoteoss/json-schema-form",
-  "version": "0.4.1-beta.0",
```

Comment:
Have you created a MR in our internal codebase to ensure the Storybook and E2E tests pass? We also need to update the docs to mention the `minDate` and `maxDate` custom keywords

Reply:
Yes, but I need to make sure I try the last changes of today

See [here](https://gitlab.com/remote-com/employ-starbase/dragon/-/merge_requests/18246)

Reply:
Yes, exactly